### PR TITLE
Parsing all documents

### PR DESCRIPTION
**Pull request by mDandini**
@Jwmazzi integrated [Apache Tika](https://tika.apache.org/) into the geoportal harvester, allowing for metadata generation of _all_ documents.

### CODE REVIEW

File: geoportal-commons/geoportal-commons-doc/src/main/java/com/esri/geoportal/commons/doc/DocUtils.java [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+import com.esri.geoportal.commons.meta.AttributeUtils;
+import com.esri.geoportal.commons.meta.MapAttribute;
+import com.esri.geoportal.commons.meta.util.WKAConstants;
+import com.esri.geoportal.commons.meta.xml.SimpleDcMetaBuilder;
+import com.esri.geoportal.commons.utils.XmlUtils;
+
+public class DocUtils {
+	
+	// Local Files For Testing Porpoises 
+	static String[] file_set = {
```
Comment (pandzel) [inline geoportal-commons/geoportal-commons-doc/src/main/java/com/esri/geoportal/commons/doc/DocUtils.java]: Are these hard coded paths necessary here?

File: geoportal-commons/geoportal-commons-doc/src/main/java/com/esri/geoportal/commons/doc/DocUtils.java [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+			"C:\\Temp\\Files\\Presentation1.pptx",
+			"C:\\Temp\\Files\\Army_Report.docx",
+			"C:\\Temp\\Files\\Scarmazzi_343971.xlsm",
+			"C:\\Temp\\Files\\TaxCollectionGRB_05.csv",
+			"C:\\Temp\\Files\\group.jpg"
+			};
+
+	// Main Used To Validate Local Files Return Bytes
+    public static void main(String[] args) throws IOException, TikaException {
```
Comment (pandzel) [inline geoportal-commons/geoportal-commons-doc/src/main/java/com/esri/geoportal/commons/doc/DocUtils.java]: I main() method necessary here?

**Comment by Jwmazzi** [comment]
No and no.

File: geoportal-commons/geoportal-commons-doc/src/main/java/com/esri/geoportal/commons/doc/DocUtils.java (hunk 1 above)
Comment (mDandini) [reply geoportal-commons/geoportal-commons-doc/src/main/java/com/esri/geoportal/commons/doc/DocUtils.java]: Removed in f63aa55

File: geoportal-commons/geoportal-commons-doc/src/main/java/com/esri/geoportal/commons/doc/DocUtils.java (hunk 2 above)
Comment (mDandini) [reply geoportal-commons/geoportal-commons-doc/src/main/java/com/esri/geoportal/commons/doc/DocUtils.java]: Removed in f63aa55

**Pull request merged**